lib/openstack_query/query_methods.py

```diff
@@ -135,60 +133,25 @@ def run(
     ):
         """
         Public method that runs the query provided and outputs
-        :param cloud_account: An string for the account from the clouds configuration to use
+        :param cloud_account: An Enum for the account from the clouds configuration to use
```

Comment:
Ditto about using the string to not limit people running locally to `Prod` / `Dev` since this is a St2 specific thing